api/test_runs.go

```diff
@@ -68,11 +47,11 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 				return
 			}
 		}
```

Comment:
It seems we need to also handle the `else` case here.

e.g. `https://test-run-filter-dot-wptdashboard-staging.appspot.com/results/?sha=61df0b90ef&complete` actuallly returns the latest run for each browser, i.e. neither `sha` nor `complete` is honored in this request.

In such case, we can either discard `complete` or return an error. WDYT?

Reply:
Switched the order so `sha` overrides it, and just ignore the complete param (erroring isn't necessary.)